.github/workflows/ci.yml

```
@@ -20,7 +20,7 @@ jobs:
     strategy:
       matrix:
         # Nextflow versions: check pipeline minimum and current latest
-        nxf_ver: ['21.04.0']
+        nxf_ver: ['21.04.0','21.10.1']
```

Any reason to specify `21.10.1`? My preference / the template leaves this as an empty string, which then tests the latest release. The intention for this test is usually to test the minimum required version of Nextflow and the latest stable release.

```suggestion
        nxf_ver: ['21.04.0','']
```

---

Thanks for the clarification!